File: pantherwatch/src/main/java/edu/gsu/pantherwatch/pantherwatch/api/MeetingsFaculty.java
package edu.gsu.pantherwatch.pantherwatch.api;
import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;

@Data
public class MeetingsFaculty {
    @JsonProperty("category")
    private String category;

    @JsonProperty("meetingTime")
    private MeetingTime meetingTime;
}

File: frontend-android/app/src/main/java/edu/gsu/pantherwatch/data/model/MeetingsFaculty.kt
package edu.gsu.pantherwatch.data.model

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class MeetingsFaculty(
    @field:Json(name = "category")
    val category: String?,
    
    @field:Json(name = "meetingTime")
    val meetingTime: MeetingTime?
)
